melos.yaml

```diff
@@ -115,7 +115,7 @@ scripts:
     packageFilters:
       published: true
 
-  format:
+  custom_format:
```

Comment:
Was `format` conflicting with some other convention?

Reply:
Yes, as of 5.1.0 `format` is a build-in command in melos. I made an issue on it [here](https://github.com/invertase/melos/issues/661), but I guess we will at most get a warning. Anyway, the default `melos format` is a bit too simple for us (currently), hence the re-name.

Reply:
Gotcha 👍